GitUp/Application/AppDelegate.m

```diff
@@ -560,7 +560,7 @@ - (IBAction)newRepository:(id)sender {
   if ([savePanel runModal] == NSFileHandlingPanelOKButton) {
     NSString* path = savePanel.URL.path;
     NSError* error;
-    if (![[NSFileManager defaultManager] fileExistsAtPath:path] || [[NSFileManager defaultManager] moveItemAtPathToTrash:path error:&error]) {
+    if (![[NSFileManager defaultManager] fileExistsAtPath:path followSymlink:NO] || [[NSFileManager defaultManager] moveItemAtPathToTrash:path error:&error]) {
```

Comment:
`-[NSFileManager moveItemAtPathToTrash:error:]` uses `-[NSFileManager moveItemAtPath:toPath:error:]` in the end, and the latter doesn't move symlink destination, only link itself is moved:

>If the last component of srcPath is a symbolic link, only the link is moved to the new path; the item pointed to by the link remains at its current location.

It looks like 2nd part of the condition doesn't follow symlinks either.